source-mongodb/pull.go

```diff
@@ -181,7 +181,14 @@ func (c *capture) ChangeStream(ctx context.Context, client *mongo.Client, bindin
 			if err = c.Output.Documents(binding, js); err != nil {
 				return fmt.Errorf("output documents failed: %w", err)
 			}
-		} else {
+		} else if ev.FullDocument != nil {
+			// FullDocument can be "null" if another operation has deleted the
```

Comment:
I'm not sure if there is anything we could do differently or better, but is it possible that the change event stream could contain an insert and then a delete for a document? In that case I assume we'd get a delete event for a document that there was never an insert event for, so it would look like something was deleted that never existed.

Reply:
@williamhbaker insert events always have a full document that is part of the event itself, this comment really only applies to updates. I will clarify this in the comment.